inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_f16.cl

```diff
@@ -129,18 +136,29 @@ KERNEL(convolution_bfyx_f16)(
                               (INPUT0_PAD_BEFORE_SIZE_X + input_x) * input_x_pitch;
 
     // Output offset calculations:
+
+
+#if OUTPUT_FORMAT_BFYX
+    const uint output_y_pitch = OUTPUT_SIZE_X;
+    const uint output_fs_pitch = output_y_pitch * OUTPUT_SIZE_Y;
```

Comment:
I thinks you just can use "OUTPUT_FEATURE_PITCH" as is, not calculating manually.

Reply:
Likewise to others too.

Reply:
fixed